scripts/ci/lib.sh

```diff
@@ -153,7 +153,9 @@ get_central_debug_dump() {
 
     require_environment "API_ENDPOINT"
     require_environment "ROX_ADMIN_PASSWORD"
-    roxctl -e "${API_ENDPOINT}" --insecure-skip-tls-verify central debug dump --output-dir "${output_dir}"
+    ROX_SERVER_NAME="" roxctl -e "${API_ENDPOINT}" \
```

Comment:
Do you know why it is necessary to empty the server name if we call with `--insecure-skip-tls-verify` anyway?

Reply:
The server name is not about insecure, but rather to tell the router the target service, so we do not ignore this parameter. In this case there is some misconfiguration between the `API_ENDPOINT` and the `ROX_SERVER_NAME`, not sure how it happened, though.

Reply:
I think I didn't answer your question. The situation needs proper understanding.